### PR TITLE
Add github_actions boilerplate

### PR DESCRIPTION
**Pull request by EngHabu**
github_actions generates two githubactions workflows to your repo:
- Pull Requests: builds and pushes images to dockerhub and github repos tagged by the git commit sha.
- Master: bumps patch-version of the repo then builds and pushes images to dockerhub & github repos tagged by that version, the git commit sha, as well as "latest"

### CODE REVIEW

**Comment by kumare3** [comment]
@EngHabu so we are going to push images for every pull request? And we are not tagging it with release?

File: boilerplate/lyft/github_workflows/master.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          WITH_V: true
+          DEFAULT_BUMP: patch
+      - name: Push Docker Image to Github Registry
+        uses: whoan/docker-build-with-cache-action@v5
+        with:
+          username: "${{ github.actor }}"
+          password: "${{ secrets.GITHUB_TOKEN }}"
+          image_name: ${{ secrets.flytegithub_repo }}/operator
```
Comment (kumare3) [inline boilerplate/lyft/github_workflows/master.yml:27]: image name will be /operator always

Comment (EngHabu) [reply boilerplate/lyft/github_workflows/master.yml:27]: Yes github docker registry requires an "imagename" after the repo name...

Comment (kumare3) [reply boilerplate/lyft/github_workflows/master.yml:27]: should it be operator

Comment (kumare3) [reply boilerplate/lyft/github_workflows/master.yml:27]: how about service etc?

**Pull request merged**